tests/subsys/lorawan/frag_decoder/testcase.yaml

```diff
@@ -0,0 +1,9 @@
+common:
+  tags:
+    - lorawan
+tests:
+  lorawan.frag_decoder:
+    integration_platforms:
```

Comment:
Does specifying `integration_platforms` when it is the only platform in `platform_allow` do anything?

Reply:
Yeah, doesn't really make sense. Removed `integration_platforms`.